src/crewai/cli/templates/crew/crew.py

```diff
@@ -1,37 +1,26 @@
 from crewai import Agent, Crew, Process, Task
-from crewai.project import CrewBase, agent, crew, task, before_kickoff, after_kickoff
-# Uncomment the following line to use an example of a custom tool
-# from {{folder_name}}.tools.custom_tool import MyCustomTool
-# Uncomment the following line to use an example of a knowledge source
-# from crewai.knowledge.source.text_file_knowledge_source import TextFileKnowledgeSource
+from crewai.project import CrewBase, agent, crew, task
 
-# Check our tools documentations for more information on how to use them
-# from crewai_tools import SerperDevTool
+# If you want to run a snippet of code before or after the crew starts, 
+# you can use the @before_kickoff and @after_kickoff decorators
+# https://docs.crewai.com/concepts/crews#example-crew-class-with-decorators
 
 @CrewBase
 class {{crew_name}}():
 	"""{{crew_name}} crew"""
 
+	# Learn more about YAML configuration files here:
+	# Agents: https://docs.crewai.com/concepts/agents#yaml-configuration-recommended
+	# Tasks: https://docs.crewai.com/concepts/tasks#yaml-configuration-recommended
```

Comment:
Wdyt about consolidating the comments to the top before the crew class?

Reply:
@lorenzejay Good idea!

Let's bring up on the daily call and let people vote!